doc/Translations.md

```diff
@@ -0,0 +1,49 @@
+# Translating Warzone 2100
+
+- [How do I help translate?](#how-do-i-help-translate)
+- [Notes for translators](#notes-for-translators)
+- [Notes for proofreaders](#notes-for-proofreaders)
+- [Notes for developers](#notes-for-developers)
+- [For developers merging translation PRs](#for-developers-merging-translation-prs)
+
+## How do I help translate?
+Help us get Warzone 2100 translated into your native language and reach more people.
+- Browse over to our Crowdin project: https://crowdin.com/project/warzone2100
+- Select the language you’d like to contribute to / review.
+   - If the language you want isn’t listed, please open a new issue at: https://github.com/Warzone2100/warzone2100/issues/new
+- If you don’t have a Crowdin account yet, you can sign up with an email, or just log-in using another account (GitHub, Google, Twitter, etc).
+- Request access to the languages you’d like to help contribute translations to!
+
+## Notes for translators:
+- All translation happens in the Crowdin project. Do not open pull requests directly on the warzone2100 repository for translation updates.
+   - This helps us ensure consistency of .po file output, provide a measure of Quality Assurance, and keep discussions about translations in a single place where they are in-line with the strings themselves.
+- By participating in this project, you agree to the code of conduct: https://github.com/Warzone2100/warzone2100/blob/master/.github/CODE_OF_CONDUCT.md
+- If you find an error in the source English strings, open a new issue / pull request on the Warzone2100/warzone2100 repository.
+- If you've been working as a translator and want to have more influence over the approved translations in your language, let us know and we'll make you a proofreader.
```

Comment:
In most cases we are lucky to get any translators at all. If more than
one person works on a translation, giving some of them a superior rank
will more likely lead to resentment among the others. Perhaps a new
contributor provides better translations than his proofreader, who only
got veto powers because he started contributing at an earlier date. And
what shall be done if a proofreader is absent?

Reply:
Crowdin encourages more collaboration around translations - someone can simply suggest a change for a single translation string. In cases where we have very solid translations already, those suggestions won't immediately/"automatically" become the new translation string - they are instead, more like a starting point for discussion.

There's a voting system that can be used for if there's a great debate / divide on which translation is most appropriate, etc.

It's probably easiest to explain project roles in Crowdin by actually testing out Crowdin. (It's not really about "veto powers".)